src/aioswitcher/api/packets.py

```diff
@@ -25,13 +25,59 @@
 
 REQUEST_FORMAT_BREEZE = "{}000001000000000000000000{}00000000000000000000f0fe"
 PAD_72_ZEROS = "0" * 72
-# format value just timestamp (initial session id is "00000000")
+P_SESSION = "00000000"
+PHONE_ID = "0000"
+DEVICE_PASS = "00000000"
+STOP_SHUTTER_PRECOMMAND = "3702"
+SET_POSITION_PRECOMMAND = "3701"
+SET_LIGHT_PRECOMMAND = "370a"
+
+# login packet for Type1 devices
+# format value just timestamp (initial session id is P_SESSION)
 LOGIN_PACKET_TYPE1 = (
-    "fef052000232a10000000000" + REQUEST_FORMAT_TYPE1[2:] + "{}" + PAD_72_ZEROS + "00"
+    "fef052000232a100"
+    + P_SESSION
```

Comment:
Are we sure this won't break?

Reply:
Yes, this is tested.. It is the same end results but with formatted differently to have more sense.

Reply:
@thecode I'm Just making sure you're ok with this before resolving.